data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,63 @@ mission "Arachi Cartoonist 2"
+			`	It's a short walk, and you end up in a building similar to a police station. The Alpha is brought before a Kimek that you guess is in charge of the facility. He asks you to give a testimony of what happened, with the Heliarch agents confirming what you say. "Your first offense in a long time this is, Alitis," the Kimek says, after checking a computer for the Alpha's data. "Three years in seclusion, your sentence is."`
+			`	After the short verdict, the agents bring the Alpha to a lower level of the building, with several empty cells, kept shut by thick, tough walls of some transparent material, with a few holes for airflow. They lock him in one of the cells, making him the only prisoner at the moment. They then ask that you wait a few minutes for a medic to arrive, to take care of your bruises.`
+			`	"Don't know why they bother with that, it's not like I hit you too hard," the Alpha says. "I guess I owe you an apology, though. I've given up on fighting humanity left and right, really, I just got a bit jumpy when I spotted you."`
+			choice
+				`	"Who are you, exactly? How did you even get here?"`
+				`	"What are you doing here in the Coalition?"`
+			`	"Where do I begin? Well, for starters, name's Alitis. Was Athis before, but I took a Saryd name after I got here. Back in the 2800s the crew I was on found some aliens somewhere far north of Earth, but we didn't get to get much from them as they were fighting another group of aliens, ones who had jump drives. We didn't want any part of the fighting, as the battle wouldn't be favorable to us, but we managed to loot a jump drive from one of the aliens. After that we started exploring disconnected systems to the east. Eventually, we joined an enclave in the Dirt Belt, but it didn't take long for the Navy to find us there. While evacuating, some of the others wanted to head back east, but I knew better, so I banked on heading further west, and eventually got here."`
+			choice
+				`	"You 'knew better?' What do you mean?"`
+				`	"What, did you have a change of heart once you got here?"`
+			`	He sits on the ground for a while, thinking. "I was one of those that fought humanity after the first Turf War. I was ambushed, cornered many times, fear wasn't a stranger to me. After we stole that jump drive and ventured east, however, someone or... something rather, some voice spoke to me. It was the first time I felt scared. After that, I wasn't inclined to go back east with the others, no matter how silly a reason they thought it was. When I got here, me and my crew helped the Heliarchs with translations for humans, and I told them about humanity and the Alphas. They took me in, but demanded my jump drive, so I handed it over, damn thing got me in more trouble than it was worth anyway."`
```

Comment:
This guy is _old_, like, are Alpha's meant to have wildly extended lifespans?

Reply:
Elias is really really old, and I don't believe Betas are mentioned to be the only ones that got super long lifespans, so I assume Alphas also have them.

Reply:
I can believe that.